network/p2p/test/sporking_test.go

```diff
@@ -87,10 +86,12 @@ func TestCrosstalkPreventionOnNetworkKeyChange(t *testing.T) {
 	require.NoError(t, err)
 
 	// create stream from node 1 to node 2
-	testOneToOneMessagingSucceeds(t, node1, peerInfo2)
+	node1.Host().Peerstore().AddAddrs(peerInfo2.ID, peerInfo2.Addrs, peerstore.AddressTTL)
+	s, err := node1.CreateStream(context.Background(), peerInfo2.ID)
+	require.NoError(t, err)
+	assert.NotNil(t, s)
```

Comment:
Why remove the encapsulation from `testOneToOneMessagingSucceeds` and have this duplicated inline in 2 places? `testOneToOneMessagingFails` is still used in 2 places.

Reply:
The test helper, comprising roughly five lines of code, served a highly specific function and was utilized in only two locations. Its generic naming did not enhance the abstraction, and the absence of reusability rendered it redundant. Therefore, we eliminated it to make way for code that is more maintainable and easier to comprehend.